packages/info/logger/mixin.server.js

```diff
@@ -0,0 +1,61 @@
+const { sync } = require('mixinable');
+const { Mixin } = require('hops-mixin');
+const { internal: bootstrap } = require('hops-bootstrap');
+
+const { callable } = sync;
+const { validate, invariant } = bootstrap;
+
+class LogBuffer {
```

Comment:
Where do we have the case, that we call `this.getLogger()` and the logger isn't there yet so we need a buffer?
I believe that shouldn't happen, because `bootstrap()` is the first hook we call https://github.com/xing/hops/blob/master/packages/react/render/mixin.server.js#L45 after `render()` https://github.com/xing/hops/blob/master/packages/react/render/index.js#L22

Reply:
And in the `constructor()`s of other mixins we don't have access to the mixin instance yet, so we can't call hooks from there.